src/app/Dice.elm

```diff
@@ -17,3 +20,33 @@ emptyCup =
 cupFromIntList : List Int -> Cup
 cupFromIntList =
     identity
+
+
+viewDie : Die -> Html msg
+viewDie die =
+    div
+        [ style
```

Comment:
Probably would be better to use Elm CSS rather than the built in one. Safer and nicer.